package.json

```diff
@@ -4,13 +4,12 @@
   "description": "Isomorphic React boilerplate",
   "main": "index.js",
   "scripts": {
-    "start": "node index",
+    "start": "APP_ENV=development node index",
     "eb-start": "npm run dist && APP_ENV=development node index",
     "dist": "NODE_ENV=production webpack --config webpack.config.js",
     "postinstall": "npm run dist",
-    "test": "node_modules/.bin/mocha test/helpers/browser.js test/unit/*.test.js",
+    "test": "node_modules/.bin/mocha test/helpers/browser.js test/unit/$TEST_FILE",
```

Comment:
Hmm that's odd. Thanks though. The other way is nice, if we can get it to work, then great.